src/index.html

```diff
@@ -164,11 +164,11 @@ <h3>About me</h3>
                     <div class="languages_section">
                         <p>Frameworks</p>
                         <ul class="languages_list">
-                            <li class="language"><p>Bootstrap</p></li>
-                            <li class="language"><p>Ruby on rails</p></li>
-                            <li class="language"><p>RSpec</p></li>
-                            <li class="language"><p>Capybara</p></li>
-                            <li class="language"><p>Selenium</p></li>
+                            <li class="language">Bootstrap</li>
```

Comment:
I wouldn't use "class" inside the li tag.  You already have a class in the ul tag, and you have a class in the parent div.  So, then for your css you could write ".languages_section > ul > li".  Perhaps that might solve the float problem we had yesterday.